tools/plot/pose_error_over_time.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+import argparse
+import logging
+import os
+import sys
+from typing import Optional
+
+import matplotlib.lines as mlines
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+from tbp.monty.frameworks.utils.logging_utils import load_stats
+
+logger = logging.getLogger(__name__)
+
+
+def pose_error_over_time(exp_path: str) -> int:
```

Comment:
Why don't these functions utilize the functions that Ramy added in `frameworks/plots/`? Are you proposing that we remove all the plotting code from Monty src and move it into the tools folder?

Reply:
Yes, that's what I'm proposing.

Reply:
okay, makes sense. Now that Ramy's code is merged into the plots folder, should this PR delete that to avoid having the same code in two places?

Reply:
I have a draft ready to go after we merge this one: https://github.com/thousandbrainsproject/tbp.monty/pull/247. I didn't want to sneak in deletions after so many approvals already.